org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/gui/custom/narratives/NarrativePanelToolbar.java

```diff
@@ -237,22 +233,20 @@ public NarrativePanelToolbar(final LiteStepControl stepControl,
     super(new FlowLayout(FlowLayout.LEFT));
 
     this._narrativeList.setModel(_narrativeListModel);
-    this._toolbarInstance = this;
     this._narrativeListModel.addColumn("");
     final TableColumn column = this._narrativeList.getColumnModel().getColumn(
         0);
     final NarrativeEntryItemRenderer narrativeEntryItemRenderer =
-        new NarrativeEntryItemRenderer();
+        new NarrativeEntryItemRenderer(model);
     column.setCellRenderer(narrativeEntryItemRenderer);
     this._narrativeList.addComponentListener(new ComponentAdapter()
     {
 
       @Override
       public void componentResized(final ComponentEvent e)
       {
-        narrativeEntryItemRenderer.setPanelWidth(_narrativeList.getWidth());
-        narrativeEntryItemRenderer.setWrapping(model.isWrapping());
-        _toolbarInstance.updateRowHeights();
+        model.setPanelWidth(_narrativeList.getWidth());
+        model.setWrapping(model.isWrapping());
```

Comment:
@saulhidalgoaular  - do we need to `setWrapping` when the component has resized?